godtools/App/Features/AppLanguage/Data/DownloadedLanguagesRepository/Cache/RealmDownloadedLanguage.swift

```diff
@@ -13,7 +13,7 @@ class RealmDownloadedLanguage: Object {
     
     @objc dynamic var createdAt: Date = Date()
     @objc dynamic var languageId: String = ""
-    @objc dynamic var downloadProgress: Double = 0
+    @objc dynamic var downloadComplete: Bool = false
```

Comment:
Hey @rachaelblue this looks good.  I didn't think we would need to track this here, but it makes sense because a user may add a language and either it fails to download or doesn't quite finish yet and we still want to track that as part of their downloadable languages.